### DISCUSSION TITLE
airflow-triggerer Liveness probe failed: No alive jobs found.

### DISCUSSION THREAD

**Discussion by chuxiangfeng** [apache/airflow#27728] [Q&A]
airflow-triggerer Liveness probe failed after a period of time: No alive jobs found. However, it does not reach the failureThreshold: 5. If the airflow-triggerer is not restarted, it indicates that the intermediate heartbeat reaches the target again. Then occasionally Liveness probe failed: No alive jobs found. How did it happen? Is it a normal phenomenon? How should I handle this?
Here is my helm-chart configuration：
`
triggerer:
  livenessProbe:
    timeoutSeconds: 120
    command:
      - sh
      - -c
      - |
        CONNECTION_CHECK_MAX_COUNT=0 AIRFLOW__LOGGING__LOGGING_LEVEL=ERROR exec /entrypoint \
        airflow jobs check --job-type TriggererJob
`
I found the same problem in scheduler before, but I found the problem because the hostname in check was wrong, which should be $(hostname -f), but I checked the trigger, which was not the problem,please help me.

airflow version：2.4.2
helm-chart：1.7.0

**Comment by potiuk** [comment]
You need to provide logs.
